spec/menu_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'menu'
+
+describe Menu do
+	it {is_expected.to respond_to :check_menu}
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing inside {.<br>Space missing inside }.<br>Tab detected.